guides/common/modules/ref_system-requirements.adoc

```diff
@@ -3,37 +3,76 @@
 [id="system-requirements_{context}"]
 = System requirements
 
-The following requirements apply to the networked base operating system:
-
-* x86_64 architecture
 ifdef::satellite[]
-* The latest version of {RHEL} 9
+{ProductName} is fully supported on both physical systems and virtual machines that run on hypervisors that are supported to run {RHEL}.
+For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization, Red Hat OpenShift Virtualization and Red Hat Enterprise Linux with KVM].
+
 endif::[]
-* 4-core 2.0 GHz CPU at a minimum
+Follow these system requirements when installing {ProductName}:
+
+* Install {ProductName} on a freshly provisioned system that servers no other function except to run {ProductName}.
+Do not use an existing system because the {Project} installer will affect the configuration of several components.
+* Ensure you have administrative user (root) access to the system.
+* Ensure the system meets the following requirements:
+** 4-core 2.0 GHz CPU or higher
 
 ifeval::["{context}" == "{project-context}"]
-* A minimum of {project-minimum-memory} RAM is required for {ProjectServer} to function.
+** {project-minimum-memory} RAM or higher
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+** 12 GB RAM or higher
+endif::[]
 ifdef::katello,satellite[]
-In addition, a minimum of 4 GB RAM of swap space is also recommended.
+** 4 GB RAM of swap space or higher
 endif::[]
-{Project} running with less RAM than the minimum value might not operate correctly.
+
+ifdef::katello,satellite[]
```

Comment:
I wonder why this is Katello-only. Please add orcharhino if it applies to Katello; or drop line 29 and 31. I assume that it does not hurt to have a valid host name for foreman-el & foreman-deb.

I favor an opinionated recommendation over a strictly technical requirement.